build_variables.bzl

```diff
@@ -785,6 +785,7 @@ libtorch_python_cuda_sources = libtorch_python_cuda_core_sources + [
 ]
 
 libtorch_python_xpu_sources = [
+    "torch/csrc/distributed/c10d/ProcessGroupXCCL.cpp",
```

Comment:
Please follow NCCL building script. This file should not be placed here. This is only a workaround.